processor/tailsamplingprocessor/README.md

```diff
@@ -428,10 +432,45 @@ As a rule of thumb, if you want to add probabilistic sampling and...
 [probabilistic_sampling_processor]: ../probabilisticsamplerprocessor
 [loadbalancing_exporter]: ../../exporter/loadbalancingexporter
 
+## Example using `processor_mode` = `decide_and_tag` with [spanmetricsconnector](../../connector/spanmetricsconnector/README.md)
+
+```yaml
+connectors:
+  forward:
+  spanmetrics:
+    exemplars:
+      enabled: true
+      dynamic:
+        enabled: true
+processors:
+  filter/drop_unsampled:
+    error_mode: ignore
+    traces:
+      span:
+      - attributes["tail_sampling.sampled"] == nil # Drop spans that are not sampled
```

Comment:
A sampling decision is trace-based, not span-based. Either a whole trace is dropped, or nothing from the trace is dropped. I understand this specific component (filter) does span filtering, but I believe adding this as the main example will confuse users.

Reply:
I would say all samplings are span based. Tail-sampling itself does not guarantee that whole trace will be sampled or not. We can never put a strong guarantee that specific trace wont be sampled but we can guarantee that a specific span wont be sampled. Also there is no way to attach decision on a trace as all processors operate on a span level anyway.

Reply:
If we have a collection of spans in memory, all of the spans belonging to the same trace will either be stored or discarded. Sure, late arriving spans might get a different sampling decision by the tail-sampling processor, but given the right conditions, the decision is consistent.

> Also there is no way to attach decision on a trace as all processors operate on a span level anyway.

If you are storing traces (collections of spans belonging to the same trace ID) in memory in the first phase, you can certainly add the sampling decision to the root span only, instead of all spans in that tree.

But I'll defer to other folks with opinions related to sampling (@jmacd and @kentquirk).